### PR TITLE
chore: new repository name, from "openvpn" to "docker-openvpn"

### PR DESCRIPTION
**Pull request by lemeurherve**

### CODE REVIEW

File: cert/pki/serial [hunk 1]
```diff
@@ -1 +1 @@
-72067956C3F94720EDC22F3A194B44BB
+b3f81fd0d189881688d07858bb7065eb
```
Comment (dduportal) [inline cert/pki/serial:1]: Where does this change come from?

Comment (lemeurherve) [reply cert/pki/serial:1]: From a mistake ^^

Comment (lemeurherve) [inline cert/pki/serial:1]:
```suggestion
72067956C3F94720EDC22F3A194B44BB
```

**Review by dduportal — APPROVED** [review]
🚀

**Pull request merged**